acoular/grids.py

```diff
@@ -721,28 +830,57 @@ def import_gpos(self):
 
 @deprecated_alias({'gpos': 'pos', 'numpoints': 'num_points'}, read_only=['gpos'])
 class LineGrid(Grid):
-    """Class for Line grid geometries."""
+    """
+    Define a 3D grid for a line geometry.
 
-    #: Staring point of the Grid
+    The :class:`LineGrid` class represents a grid where points are arranged linearly in 3D space.
+    The grid is defined by a starting location (:attr:`loc`), a direction vector
+    (:attr:`direction`), a total length (:attr:`length`), and the number of points
+    (:attr:`num_points`) along the line.
+
+    Notes
+    -----
+    - The distance between points is :attr:`length` ``/ (`` :attr:`num_points` ``- 1)``.
+    - The direction vector is normalized to ensure consistency.
+
+    Examples
+    --------
+    Create a line grid with 5 points along the x-axis, starting at (0, 0, 0), with a length of 4
+    meters:
+
+    >>> import acoular as ac
+    >>> grid = ac.grids.LineGrid()
+    >>> grid.loc = (0.0, 0.0, 0.0)
+    >>> grid.direction = (1.0, 0.0, 0.0)
+    >>> grid.length = 4
+    >>> grid.num_points = 5
+    >>> grid.pos
+    array([[0., 1., 2., 3., 4.],
+           [0., 0., 0., 0., 0.],
+           [0., 0., 0., 0., 0.]])
+    """
+
+    #: Starting point of the grid in 3D space. Default is (0.0, 0.0, 0.0).
     loc = Tuple((0.0, 0.0, 0.0))
 
-    #: Vector to define the orientation of the line source
+    #: A vector defining the orientation of the line in 3D space. Default is (1.0, 0.0, 0.0).
     direction = Tuple((1.0, 0.0, 0.0), desc='Line orientation ')
 
-    #: Vector to define the length of the line source in meter
+    #: Total length of the line in meters. Default is 1.0.
```

Comment:
The values are not necessarily in meters. It depends on what the user defines as the unit of length.
```suggestion
    #: Total length of the line. Default is 1.0.
```